src/tmpfiles/tmpfiles.c

```diff
@@ -2753,9 +2838,15 @@ static int create_item(Item *i) {
         return 0;
 }
 
-static int remove_item_instance(Item *i, const char *instance, CreationMode creation) {
+static int remove_item_instance(
+                Context *c,
+                Item *i,
+                const char *instance,
+                CreationMode creation) {
+
         int r;
 
+        assert(c);
```

Comment:
not used here either

Reply:
similar, we want to pass this function as param to glob_item()